tests/python/frontend/pytorch/test_forward.py

```diff
@@ -227,13 +230,14 @@ def verify_model_with_input(
 
             compiled_output = relay_model.get_output(0).numpy()
             assert_shapes_match(baseline_outputs, compiled_output)
-            if assert_shape_only == False:
+            if assert_shape_only is False:
                 tvm.testing.assert_allclose(baseline_outputs, compiled_output, rtol=rtol, atol=atol)
 
 
 # Single operator tests
 @tvm.testing.uses_gpu
 def test_forward_pixel_shuffle():
+    """PixelShuffle"""
```

Comment:
per the previous discussion, can you adjust the docstrings to match the function name?

Reply:
Ok！

Reply:
Just to ensure, adjust the docstrings like this `Test forward pixel shuffle`?

Reply:
er i was thinking just use an exact copy of the test name: `test_forward_pixel_shuffle`. Then we don't create two atoms to grep for.